cl/opinion_page/templates/view_opinion_summaries.html

```diff
@@ -68,17 +68,20 @@ <h3>{{ summaries_count|intcomma }} judge-written summar{{ summaries_count|plural
               <ul class="group-summaries collapse" id="group-summaries-{{ representative.pk }}">
                 {% for summary in group.parentheticals.all %}
                   {% with describing_cluster=summary.describing_opinion.cluster %}
-                    <li class="v-offset-below-1">
-                      {{ summary.text|capfirst }}
-                      <br/>
-                      <span class="bullet-tail">{{ describing_cluster.date_filed }}</span>
-                      <span class="bullet-tail">
+                    {# Don't duplicate the representative cluster in the nested group #}
+                    {% if describing_cluster != representative_cluster %}
```

Comment:
I assume you tested this and it works properly but it feels a bit sketchy to do an object comparison here instead of comparing primary keys. If this is Django convention, though, then feel free to disregard.

Reply:
Yep, Django makes it safe: https://docs.djangoproject.com/en/4.0/topics/db/queries/#comparing-objects